moesifwsgi/config_manager.py

```diff
@@ -54,11 +72,12 @@ def update_configuration(self, response_etag):
             # We need to check the etag again because it might have changed while we were waiting for the lock.
             # If there was an unrecoverable failure in this update call, new_etag will be None, 
             # and saving this value without updating the configuration will cause us to retry the update on the next request.
-            if new_etag != self.current_etag:
-                self.current_etag = new_etag
-                if config is not None:
-                    self.config = config
-                    self.last_updated_time = new_last_updated_time
+            # if new_etag != self.current_etag:
+            self.current_etag = new_etag
+            if config is not None:
+                self.config = config
+                self.last_updated_time = new_last_updated_time
+                logger.debug("config update at " + str(self.last_updated_time))
```

Comment:
log this only incase of debug flag is set to avoid printing too much log

Reply:
Done